.github/workflows/hugo.yml

```diff
@@ -56,8 +56,8 @@ jobs:
         run: |
           cd site
           hugo \
-            --minify \
-            --baseURL "${{ steps.pages.outputs.base_url }}/"
```

Comment:
baseURL is required

Reply:
yes. I've added two environment config for baseurl.

Reply:
hugo will load config under githubpage when --environment githubpage is specified. 
and config.toml under githubpage will set baseurl to  "https://kubernetes-sigs.github.io/cloud-provider-azure/"